pkg/config/defaults.go

```diff
@@ -112,6 +113,8 @@ func setDefaults(cfgType string) {
 	viper.SetDefault("database.dynamodb.table_name", "kvstore")
 	viper.SetDefault("database.dynamodb.scan_limit", 1024)
 
+	viper.SetDefault("database.cosmosdb.throughput", DefaultCosmosDBThroughput)
```

Comment:
Suggest to remove the default and have an option not to configure throughput.
It will also keep the current behaviour.